### PR TITLE
Changed modals to divs

### LINKED ISSUE #142: Refactor to use less modals
Changing the coin animation and the player turn animation to no longer be in modals, seems to be causing some of the choppiness when they come and go. Also if the timing of the modals are out of sync it seems to cause the screen shrinking bug and probably other unwanted behaviour. Maybe use divs in the main page with dynamic display options (none or block when we want it to appear).

### PR DESCRIPTION
**Pull request by vincentcote87**
Fixes #142 
#### Overview of changes

- Changed the player turn and active side modals to simply use divs with fade transitions.
- Lowers the chance of having overlapping modals which leads to undesired results.
-

Reviewer: @johnanvik

### CODE REVIEW

File: src/components/PlayerInfoPanel.vue [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
     <div id="player-info-panel">
 
-      <div id="playerTurn" class="modal fade yourTurn" tabindex="-1" role="dialog" aria-labelledby="" aria-hidden="true">
-        <div class="modal-dialog" role="document">
-          <div class="modal-content" style="border-radius: 30px">
-            <h4 class="modal-title">{{ currentPlayerName }}, It's Your Turn</h4>
-          </div>
-        </div>
-      </div>
+      <!--<div id="playerTurn" class="modal fade yourTurn" tabindex="-1" role="dialog" aria-labelledby="" aria-hidden="true">-->
```
Comment (johnanvik) [inline src/components/PlayerInfoPanel.vue]: Commented out? Should this be removed?

File: src/store/mutations.js [hunk 2]
```diff
@@ -236,13 +236,15 @@ export default {
...
-  playerModalTrigger() {
-    $('#playerTurn').modal('handleUpdate')
-    $('#playerTurn').modal('show');
-  },
-  playerModalHide() {
-    $('#playerTurn').modal('handleUpdate')
-    $('#playerTurn').modal('hide');
+  playerModalTrigger(state) {
+    state.playerTurn = true;
+    // $('#playerTurn').modal('handleUpdate')
```
Comment (johnanvik) [inline src/store/mutations.js]: Commented out code

File: src/store/mutations.js [hunk 3]
```diff
@@ -236,13 +236,15 @@ export default {
...
+    // $('#playerTurn').modal('show');
+  },
+  playerModalHide(state) {
+    state.playerTurn = false;
+    // $('#playerTurn').modal('handleUpdate')
```
Comment (johnanvik) [inline src/store/mutations.js]: Commented out code

**Comment by vincentcote87** [comment]
I just pulled the latest of this branch and these commented code seems to have been removed already.  As far as functionality it should look the same but uses divs instead of modals for these components which shouldn't use modals.

**Pull request merged**